cob/cob.c

```diff
@@ -831,8 +831,8 @@ int m0_cob_domain_destroy(struct m0_cob_domain *dom,
 	m0_buf_init(&key, &omgkey, sizeof omgkey);
 	cob_table_delete(dom->cd_fileattr_omg, tx, &key);
 
-	m0_cob_nskey_make(&nskey, &M0_COB_ROOT_FID, M0_COB_ROOT_NAME,
-			  strlen(M0_COB_ROOT_NAME));
+	rc = m0_cob_nskey_make(&nskey, &M0_COB_ROOT_FID, M0_COB_ROOT_NAME,
+			       strlen(M0_COB_ROOT_NAME));
```

Comment:
do you want to check for rc != 0, otherwise rc will get overwritten by line 836 m0_cob_lookup.

Reply:
actually i do not need to get the return value as it was before. As a part of other fix this code remained. I will remove the assignment to rc.